TorchSharp/TypeGeneration.tt

```diff
@@ -769,7 +851,43 @@ foreach (var type in new [] {
 
 <#          } /* foreach */ } #>
 
+<#
+        // Idiom Op (Result, T, Scalar, Source2)
+            if (isFloat){
+#>                
         [DllImport ("caffe2")]
+        extern static void TH<#=tname#>Tensor_pow (HType result, HType x, <#=ttype#> y);
+
+        /// <summary>
+        ///   Returns a new tensor with <see paramref="this"/> raised to the power of <see paramref="y"/>.
+        /// </summary>
+		/// <param 
+        /// <param name="y">The exponent.</param>
+        public <#=tname#>Tensor Pow (<#=ttype#> y)
```

Comment:
We would have to come up with a naming convention for that. Trailing underscore is kind of ugly, but I agree that it might be rather useful. Maybe a separate effort to do that systematically?

Reply:
There are already some gaps in the API for those conveniences.

We can address those later.